Gordon360/Models/ViewModels/HousingApplicantViewModel.cs

```diff
@@ -0,0 +1,19 @@
+using Gordon360.Models.CCT;
+using Gordon360.Models.ViewModels.RecIM;
+
+namespace Gordon360.Models.ViewModels
+{
+    public class HousingApplicantViewModel
```

Comment:
Isn't this the same as the database model? Do we need a separate ViewModel for it?

Reply:
Yeah I guess you are right. I created these view models is because I was worried that sending unnecessary information, such as active and rowID, to the frontend would confuse it, but after testing, it seems like the frontend works fine with this information, so I think I can change the view models to database models.